tests/test_modeling_common.py

```diff
@@ -3590,6 +3592,8 @@ def get_mean_reldiff(failcase, x, ref, atol, rtol):
                 model.save_pretrained(tmpdirname)
                 model_sdpa = model_class.from_pretrained(tmpdirname, torch_dtype=torch_dtype)
                 model_sdpa = model_sdpa.eval().to(torch_device)
+                if hasattr(model_sdpa.config, "vision_config"):
+                    self.assertTrue(model_sdpa.config.vision_config._attn_implementation == "sdpa")
```

Comment:
We shouldn't need to do this - adding this to the model is an indication it's not being set correctly within the model

Reply:
Done in e78108b4c.